prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -279,15 +281,14 @@ private void UpdateMailingAddress(Enrollee dbEnrollee, MailingAddress newAddress
                 _context.Remove(item);
             }
 
+            if (newCollection == null) return;
```

Comment:
I prefer using braces even for one-liner if statements

Reply:
Haha yah same.  That's an IDE update

Reply:
Find another as well.  Will update Rider IDE to not remove brackets during formatting.